azure-mgmt-servicebus/src/main/java/com/microsoft/azure/management/servicebus/implementation/TimeSpan.java

```diff
@@ -395,7 +395,7 @@ String getRawValue() {
     }
 
     boolean isTerminalMatched(Character charToMatch) {
-        return terminalChar == charToMatch;
```

Comment:
Fixing incorrect Character comparison

Reply:
I think this is the bug that is causing the Travis CI to fail... Just like the other comment, please do a check for terminalChar != null && charToMatch != null